tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "allowSyntheticDefaultImports": true,
     "esModuleInterop": true,
     "isolatedModules": true,
-    "jsx": "react-native",
+    "jsx": "react",
```

Comment:
Just to get a deeper understanding, may I ask why this change is necessary? 🙂

Reply:
I found out that `ts-jest` can't recognize JSX/TSX when this is set to `react-native`. You can easily reproduce it by switching it back to `react-native` and running `npm test` / `yarn test`, it will throw a `unexpected Token <`-error. I really appreciate your interest 😄

Reply:
Oh great, I've had that exact problem before, I din't know it had anything to do with that setting. I'll have to try it out tomorrow!